active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
+
+Delegated responsibilities:
+
+- Contacting, developing and implementing financial sponsorships with donor sources
+- Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django
```

Comment:
Rewording so it’s not just about donations (unless you do intend for it to be). I wonder if it’d be worth even being more specific and calling out different types of funding directly.

```suggestion
- Contacting, developing and implementing financial sponsorships with funding sources
- Clarifying and elaborating on the various creative strategies that funding sources of different sizes and types can use to support Django
```